### PR TITLE
161 spacing megathread

### PR DESCRIPTION
**Pull request by ikopke23**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [x] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Linking this to the issue 161 that I previously opened saying that section headers would exist and then the actual text would start on the same line.
I should have fixed that in all of the cases that I found.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -432,8 +436,6 @@
 \\*\\*
 There is one permanent directorship for each major aspect of the government of the House and each one is chaired by an E-Board member.
 A voting E-Board member is considered to be any person with a non-zero vote on E-Board. % Defined this way so that (for example) each member in a dual directorship is each considered to be a voting member on E-Board.
-Ad Hoc directorships are created on an as-needed basis.
-They are generally very task oriented and are chaired by a House member.
```
Comment (mxmeinhold) [inline constitution.tex:436]: This looks like an unrelated change?

File: constitution.tex [hunk 2]
```diff
@@ -610,12 +606,12 @@ \subsubsection{Responsibilities of the R\&D Director}
 
 \asubsection{Selection}
 \asubsubsection{Dual Directorship}
+\hfill\break\hfill\break
```
Comment (mxmeinhold) [inline constitution.tex:609]: If you're doing this for every subsubsection, it's probably better to adjust the definition of asubsubsection rather than all the copy pasting

Comment (ikopke23) [reply constitution.tex:609]: I thought of that but I don't know how to do that

File: constitution.tex (hunk 1 above)
Comment (ikopke23) [reply constitution.tex:436]: It is I am not entirely sure what happened there

Comment (ikopke23) [reply constitution.tex:436]: I figured out what happened. I change which branch  I was working in and then I needed to pull to get a fresh copy of the main document to work on

File: constitution.tex (hunk 2 above)
Comment (ikopke23) [reply constitution.tex:609]: If you know how to do that you could let me know how or where to find it

Comment (mxmeinhold) [reply constitution.tex:609]: It's defined here https://github.com/rhochgraf21/Constitution/blob/ca316870446c2005d222596034f85c04232179e6/constitution.tex#L45

Comment (ikopke23) [reply constitution.tex:609]: thank you
